package.json

```diff
@@ -44,14 +44,14 @@
 		"jest": "^26.1.0",
 		"npm-run-all": "^4.1.5",
 		"prettier": "^2.0.5",
-		"runtypes": "^5.2.0",
+		"runtypes": "^6.0.0",
 		"ts-jest": "^26.1.3",
 		"tslint": "^6.1.2",
 		"tslint-config-airbnb": "^5.11.2",
 		"tslint-config-prettier": "^1.18.0",
-		"typescript": "^3.9.7"
+		"typescript": "^4.2.3"
 	},
 	"peerDependencies": {
-		"runtypes": "^5.0.1"
+		"runtypes": ">= 5 < 7"
```

Comment:
Node semver appears to parse this fine, but can you make this `>=5 <7`to match typical formatting?

Thanks!